public/scripts/instruct-mode.js

```diff
@@ -256,7 +256,12 @@ export function formatInstructModeChat(name, mes, isUser, isNarrator, forceAvata
 export function formatInstructModeSystemPrompt(systemPrompt){
     if (power_user.instruct.system_sequence) {
         const separator = power_user.instruct.wrap ? '\n' : '';
-        return power_user.instruct.system_sequence + separator + systemPrompt;
+
+        if (power_user.instruct.system_sequence.includes("{{sys}}")) {
+            return power_user.instruct.system_sequence.replace(/{{sys}}/gi, systemPrompt) + separator;
```

Comment:
The [official Llama 2 Chat prompt format](https://huggingface.co/blog/llama2#how-to-prompt-llama-2) has an empty line between the system prompt and user message. That's what I'm getting with this.

Reply:
Just FYI and to document this: I didn't add the `<s>` and `</s>` strings you had in your example image on the other PR. The `<s>` is the BOS (Begin of Sequence) token that the inference software is supposed to add by itself, and the `</s>` is the EOS (End of Sequence) token that the model is supposed to output to signal end of inference. And both are special tokens, not the actual string we see here, so they can't be used in the sequences anyway. (I only add the `</s>` to my stopping strings because I've seen some models that were incorrectly trained or had tokenization bugs so they output the literal string instead of the special token.)

Reply:
> Is a separator at the end really necessary? With new sys template it would be two newlines at the end

Thinking about this some more, maybe you're right! Do you think it would be better to add an additional newline to the system sequence instead of adding the seperator here?